File: 2.0/python/campaigns/getCampaignSound.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.campaigns.getCampaignSound(
    id=379506003,
    # return only specific fields
    fields='name,lengthInSeconds'
).result()

# see sample JSON response for this API
# on 'curl' samples tab
print(response)

File: 2.0/php/campaigns/getCampaignSound.php
<?php

class ApiClientSample{

    public static function main(){
        $client = \CallFire\Api\DocumentedClient::createClient("login", "password");
        $request = $client->getCampaignSound();
        $request->getOperationConfig()->setPathParameters(array("id" => 12345));
        $request->getOperationConfig()->setQueryParameters(array("fields" => "name,status,lengthInSeconds"));
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();

